pkgs/by-name/ds/dstask/package.nix

```
@@ -1,29 +1,50 @@
 {
   lib,
+  pkgs,
+  stdenv,
   buildGoModule,
   fetchFromGitHub,
+  installShellFiles,
 }:
 
 buildGoModule rec {
   pname = "dstask";
-  version = "0.26";
+  version = "0.27";
+
+  nativeBuildInputs = [
+    installShellFiles
+    pkgs.git # system dependency, even for completion generation commands
```

I'm not sure quite what we need for this but we shouldn't add `pkgs` and I suggets `gitMinimal` due to closure size
```suggestion
    gitMinimal # system dependency, even for completion generation commands
```

---

I'll try this, and add another comment that indicates that it can be removed for next version because https://github.com/naggie/dstask/pull/197 was merged.

---

Adding pkgs is a bad practice beause it may create some circular dependancy?

---

Not 100% sure but I would say that it makes `.override` possible.